Classes/Domain/Model/Index.php

```diff
@@ -152,14 +155,14 @@ public function exists()
      * @param string $path
      * @param array $arguments
      * @param string $content
-     * @param boolean $prefixIndex
+     * @param bool $prefixIndex
      * @return Response
-     * @throws ElasticSearchException
+     * @throws \Exception
      */
-    public function request($method, $path = null, array $arguments = [], $content = null, $prefixIndex = true)
+    public function request(string $method, ?string $path = null, array $arguments = [], ?string $content = null, bool $prefixIndex = true): Response
```

Comment:
`?string $foo = null` is not needed, if `null` is given as default value, the "nullable modifier" is not needed. And vice versa, AFAIK.